iphone/Classes/MediaModule.m

```diff
@@ -1784,7 +1784,7 @@ - (void)handleTrimmedVideo:(NSURL *)theURL withDictionary:(NSDictionary *)dictio
 }
 #endif
 
-#if IS_SDK_IOS_14
+#if IS_SDK_IOS_14 && defined(USE_TI_MEDIAOPENPHOTOGALLERY)
```

Comment:
You _may_ also need to check it's delegate methods, but not sure if they're also affected.

Reply:
delegate methods appear to already be guarded